### PR TITLE
Press page font color fixed

### LINKED ISSUE #613: Press page is hiding the format of the file with a black font
screen shot 2018-12-14 at 09 30 57

You can't see the `SVG` or `PNG` because it's too dark.

### PR DESCRIPTION
**Pull request by Rodriq**
Fixing font color on press page _Logos and Assets area_
Changing from dark to white "Fixes #613"
@rafaelks check to confirm.

color-change

### CODE REVIEW

File: install.html [hunk 1]
```diff
@@ -75,6 +75,9 @@ <h2 class="display theme_type--dark">Mobile Apps</h2>
         <a target="_blank" rel="noopener noreferrer" href="//itunes.apple.com/app/rocket-chat/id1148741252">
           <img class="itunes" src="/images/install/itunes-store.svg">
         </a>
+        <a target="_blank" rel="noopener noreferrer" href="//f-droid.org/en/packages/chat.rocket.android/">
```
Comment (MartinSchoeler) [inline install.html]: These changes seems unrelated to the pull request, can you remove them?

Comment (Rodriq) [reply install.html]: Ohh @MartinSchoeler  Didn't noticed this commit was pushed here.
  Sure I'll do the amendment.
